app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ReviewsController < ApplicationController
+  # This controller is nested - its routes are under /restaurants/:restaurant_id/...
+
+  before_action :query_review, only: [:show, :update, :destroy]
+
+  def index
+    render json: Review.where(restaurant_id: params.require(:restaurant_id)).all
+  end
+
+  def show
+    render json: @review
+  end
+
+  def create
+    @review = Review.new(review_params)
+    @review.restaurant = params[:restaurant_id]
```

Comment:
I think you should be able to pass restaurant_id to Review.new, as long as it's defined as a foreign key. 
I think in the migration you should have: "t.belongs_to :restaurant, foreign_key: true"

Reply:
`Restaurant.find(params[:restaurant_id]).create_review(review_params)`